MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -208,189 +345,80 @@ public string BaseUrl
+        public bool RequireHttps { get; set; } = false;
 
-        public bool EnableNewOmdbSupport { get; set; }
+        public bool EnableNewOmdbSupport { get; set; } = true;
 
-        public string[] RemoteIPFilter { get; set; }
+        /// <summary>
+        /// Gets or sets the filter for remote IP connectivity. Used in conjuntion with <seealso cref="IsRemoteIPFilterBlacklist"/>.
+        /// </summary>
+        public string[] RemoteIPFilter { get; set; } = Array.Empty<string>();
 
-        public bool IsRemoteIPFilterBlacklist { get; set; }
+        /// <summary>
+        /// Gets or sets a value indicating whether <seealso cref="RemoteIPFilter"/> contains a blacklist or a whitelist. Default is a whitelist.
+        /// </summary>
+        public bool IsRemoteIPFilterBlacklist { get; set; } = false;
 
-        public int ImageExtractionTimeoutMs { get; set; }
+        public int ImageExtractionTimeoutMs { get; set; } = 0;
 
-        public PathSubstitution[] PathSubstitutions { get; set; }
+        public PathSubstitution[] PathSubstitutions { get; set; } = Array.Empty<PathSubstitution>();
 
-        public bool EnableSimpleArtistDetection { get; set; }
+        public bool EnableSimpleArtistDetection { get; set; } = false;
 
-        public string[] UninstalledPlugins { get; set; }
+        public string[] UninstalledPlugins { get; set; } = Array.Empty<string>();
 
         /// <summary>
         /// Gets or sets a value indicating whether slow server responses should be logged as a warning.
         /// </summary>
-        public bool EnableSlowResponseWarning { get; set; }
+        public bool EnableSlowResponseWarning { get; set; } = true;
 
         /// <summary>
         /// Gets or sets the threshold for the slow response time warning in ms.
         /// </summary>
-        public long SlowResponseThresholdMs { get; set; }
+        public long SlowResponseThresholdMs { get; set; } = 500;
 
         /// <summary>
         /// Gets or sets the cors hosts.
         /// </summary>
-        public string[] CorsHosts { get; set; }
+        public string[] CorsHosts { get; set; } = new[] { "*" };
 
         /// <summary>
         /// Gets or sets the known proxies.
         /// </summary>
-        public string[] KnownProxies { get; set; }
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="ServerConfiguration" /> class.
-        /// </summary>
-        public ServerConfiguration()
-        {
-            UninstalledPlugins = Array.Empty<string>();
-            RemoteIPFilter = Array.Empty<string>();
-            LocalNetworkSubnets = Array.Empty<string>();
-            LocalNetworkAddresses = Array.Empty<string>();
-            CodecsUsed = Array.Empty<string>();
-            PathSubstitutions = Array.Empty<PathSubstitution>();
-            IgnoreVirtualInterfaces = false;
-            EnableSimpleArtistDetection = false;
-            SkipDeserializationForBasicTypes = true;
```

Comment:
Is there a benefit to moving these defaults? I always thought it was much easier to look everything up when it was in one location, but it's not that important if we document the settings somewhere else.

Reply:
Either is good, as long as we all stick to the same method, but we don't seem to have a standard anywhere.

Personally, I prefer this method in the configs, as i tend to go looking for the name, as i can't remember it exactly - then have to go and find the settings.